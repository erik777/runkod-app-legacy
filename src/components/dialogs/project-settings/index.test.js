import React from 'react';
import {DialogContent} from './index';
import TestRenderer from 'react-test-renderer';

const fnProps = {
  selectProject: () => {
  },
  fetchProjects: () => {
  },
  setProjectStatus: () => {
  },
  toggleUiProp: () => {
  },
};


test('1- default', () => {
  const props = Object.assign({}, {
    project: {
      _id: 'unique_id',
      name: 'test.com',
      status: 1,
    },
  }, fnProps);

  const renderer = TestRenderer.create(
    <DialogContent {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});

