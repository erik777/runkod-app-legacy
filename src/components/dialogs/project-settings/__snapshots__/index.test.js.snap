// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default 1`] = `
<div
  className="setting-sections"
>
  <div
    className="setting-section"
  >
    <div
      className="title"
    >
      Status
       
      <span
        className="project-status status-on"
      >
        On
      </span>
    </div>
    <div
      className="description"
    >
      Control status of this project.
    </div>
    <button
      className="btn btn-outline-primary btn-sm"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Change
    </button>
  </div>
  <div
    className="setting-section"
  >
    <div
      className="title"
    >
      Delete this project
    </div>
    <div
      className="description"
    >
      This action might not be undone. Please be certain.
    </div>
    <button
      className="btn btn-outline-danger btn-sm"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Delete
    </button>
  </div>
</div>
`;
